### PR TITLE
Bump CoreDNS version to 1.2.4 and update manifest

### PR DESCRIPTION
**Pull request by rajansandeep**
Updates CoreDNS version to 1.2.4 and updates the manifest.

### CODE REVIEW

File: upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.6.yaml.template [hunk 1]
```diff
@@ -55,7 +61,6 @@ data:
   Corefile: |
     .:53 {
         errors
-        log
```
Comment (rochacon) [inline upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.6.yaml.template]: ❤️  Thank you!! Logging every DNS request is so verbose, I've been removing this manually on every cluster.

**Comment by justinsb** [comment]
Thanks @rajansandeep - particularly for remembering to bump bootstrapchannelbuilder :-)

/approve

**Pull request merged**